cluster/seq-cluster-docker-compose/compose.yaml

```diff
@@ -2,20 +2,25 @@ services:
   seq1:
     image: datalust/seq:preview
     hostname: seq1
+    depends_on:
+      pgseq:
+        condition: service_healthy
     networks:
       - seqclusterdemo
     volumes:
       - ./seq1data:/data
     environment: 
       - ACCEPT_EULA=Y
       - "SEQ_METASTORE_POSTGRES_CONNECTIONSTRING=Server=pgseq;Port=5432;Database=seq;User Id=postgres;Password=terminatorx;"
-      - SEQ_API_CANONICALURI=http://proxy:5666
+      - SEQ_API_CANONICALURI=http://localhost:5666
       - SEQ_CLUSTER_NODENAME=alpha
       - SEQ_CLUSTER_ISENABLED=true
       - SEQ_CLUSTER_CLUSTERLISTENURI=ws://seq1:1082
       - SEQ_CLUSTER_INTERNALAPIURI=http://seq1
       - 'SEQ_STORAGE_SECRETKEY=lON57A+lulu9J1RKf8ft03FtOpepXJldYtsGLybHHP4='
-      - 'SEQ_DIAGNOSTICS_INTERNALLOGSERVERURL=http://seqdiagnostics'
+      - SEQ_DIAGNOSTICS_OTLP_ENDPOINT=http://seqdiagnostics/ingest/otlp
+      - SEQ_DIAGNOSTICS_OTLP_PROTOCOL=http/protobuf
+      - SEQ_DIAGNOSTICS_OTLP_METRICSSAMPLINGINTERVALSECONDS=5
```

Comment:
The `SEQ_DIAGNOSTICS_OTLP_*` variables replace the `SEQ_DIAGNOSTICS_INTERNALLOGSERVER*` ones, when they're used.